src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const lines = sourceString.split(';');
+  const cssStyles = {};
```

Comment:
The variable name `cssStyles` is clear and descriptive, which is good. However, it's important to ensure that the variable names you choose are consistent and meaningful throughout your code.